processing/src/main/java/io/druid/query/DefaultQueryMetrics.java

```diff
@@ -66,15 +65,6 @@ public void queryType(QueryType query)
   }
 
   @Override
-  public void interval(QueryType query)
```

Comment:
Oh, I realized that QueryMetrics is added recently and not release yet which means there's no compatibility issue. You mean just many users want to filter metrics by data source alone without intervals?